_posts/2018-13-06-announcing-audit-tool.markdown

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title:  "Announcing Gemfile.lock Audit Tool"
+date: 2018-06-13 11:10:00
+categories: ["rails", "ruby"]
+author: "emily"
+---
+
+Today we are happy to announce the launch of our new microsite: [Gemfile.lock Audit Tool](https://audit.fastruby.io) - a tool created to allow users to check their Gemfile.lock for vulnerabilities in a quick and secure manner.
+
+<!--more-->
+
+The tool uses the [bundler-audit gem](https://github.com/rubysec/bundler-audit) to check for vulnerable versions of gems and insecure gem sources. The tool updates automatically with new warnings as the [bundler-audit gem](https://github.com/rubysec/bundler-audit) database of vulnerabilities is updated.
```

Comment:
I would only add the link to bundler-audit gem the first time. But this works too 👍